src/server/editionsPage.tsx

```diff
@@ -0,0 +1,79 @@
+// ----- Imports ----- //
+
+import React from 'react';
+import { renderToString } from 'react-dom/server';
+import { JSDOM } from 'jsdom';
+import { RenderingRequest } from '@guardian/apps-rendering-api-models/renderingRequest';
+import { none, Option } from '@guardian/types/option';
+
+import { fromCapi } from 'item';
+import Article from 'components/editions/article';
+import { pageFonts } from 'styles';
+
+
+// ----- Types ----- //
+
+interface Page {
+    html: string;
+    clientScript: Option<string>;
+}
+
+
+// ----- Setup ----- //
+
+const docParser = JSDOM.fragment.bind(null);
+
+
+// ----- Functions ----- //
+
+const renderHead = (request: RenderingRequest): string =>
+    renderToString(
+        <>
+            <meta charSet="utf-8" />
+            <title>{request.content.webTitle}</title>
+            <meta name="viewport" content="initial-scale=1" />
```

Comment:
I think we set a max scale on apps-rendering to avoid users zooming in, could this be an issue on editions?

Reply:
Out of interest, why do we prevent users zooming?

Reply:
Good question, It's been like that for a while in the templates, I'm not sure when the original decision was made but maybe preventing zoom made the articles feel more "native"?

We have the font resize feature on apps which works nicely defining everything in rem. It doesn't work for all cases though e.g. cartoons with text 

First article is templates and second is apps-rendering
![80618203-ef8db800-8a3a-11ea-9cac-be5937ed7ccc](https://user-images.githubusercontent.com/11618797/97206063-29543700-17b0-11eb-994c-0f2779fe387a.gif)

Reply:
I was going to ask this too actually. The `maximum-scale` limit was flagged by Lighthouse as an accessibility problem so I didn't include it here. I have two questions on this:

1. Why do we include it at the moment in Apps-Rendering?
2. Does it make a difference considering we allow users to zoom everything on the page with the slider? Does it respect the system zoom level?